playbooks/openstack/openshift-cluster/files/heat_stack.yaml

```diff
@@ -350,7 +350,6 @@ resources:
           port_range_min: 10250
           port_range_max: 10250
           remote_mode: remote_group_id
-          remote_group_id: { get_resource: master-secgrp }
```

Comment:
Does this change the security access to the nodes group?

I'm almost inclined to say that we should limit it to "infra" nodes only.